### ISSUE TITLE
Quicksend/quicksave only to whitelisted trusted devices

### ISSUE THREAD

**Issue by c4vr** [localsend/localsend#845]
### Is your feature request related to a problem? Please describe.

Currently quicksave allows any device on the local network to send files to me. I want to allow the quicksave, but restrict which devices are allowed to send the files.

### Describe the solution you'd like

Add an option to create a "whitelist" of trusted devices or maybe add an option to the favorites list to allow those devices to "quicksend" files to you, while making sure that other people can't impersonate those trusted devices.

**Comment by adhisimon** [comment]
Yes, it's a nice feature. Other issue mentioning it: #655 and #463

**Comment by nopeless** [comment]
seconded
